src/Xamarin.Android.Build.Tasks/Tasks/BuildAppBundle.cs

```diff
@@ -119,7 +122,11 @@ protected override CommandLineBuilder GetCommandLineBuilder ()
 		{
 			var cmd = base.GetCommandLineBuilder ();
 			cmd.AppendSwitch ("build-bundle");
-			cmd.AppendSwitchIfNotNull ("--modules ", BaseZip);
+			var modules = new List<string> ();
+			modules.Add (BaseZip);
+			if (Modules != null && Modules.Any ())
+				modules.AddRange (Modules);
+			cmd.AppendSwitchIfNotNull ("--modules ", string.Join (",", modules));
```

Comment:
We might hit command line argument length issues in the future with this if we are not careful.